command/v7/shared/app_summary_displayer_test.go

```diff
@@ -698,10 +698,58 @@ var _ = Describe("app summary displayer", func() {
 		When("there is an active deployment", func() {
 			var LastStatusChangeTimeString = "2024-07-29T17:32:29Z"
 			var dateTimeRegexPattern = `[a-zA-Z]{3}\s\d{2}\s[a-zA-Z]{3}\s\d{2}\:\d{2}\:\d{2}\s[A-Z]{3}\s\d{4}`
+			var maxInFlightDefaultValue = 1
```

Comment:
thought (non-blocking): Should we keep constants in a single place [similar to this](https://github.com/pivotalgeorge/cf-cli/blob/ffa3f52edbee13a40228c66c3f1327a90020ca8c/integration/helpers/constants.go#L3-L4)

Reply:
+1 to @joaopapereira's different PR suggestion if we do this. also, I don't think we should extract here because:
- `LastStatusChangeTimeString` and `dateTimeRegexPattern` (and not `maxInFlightDefaultValue`) are the type of helper constants that belong as siblings in the config @a-b linked.
- the three vars here are related to the tests in scope
- we'd be making readers jump around if we break them apart

In general I think we should extract when we see a need for reuse. It's unfortunate that the regex here is off-by-one from [this one in constants.go](https://github.com/pivotalgeorge/cf-cli/blob/ffa3f52edbee13a40228c66c3f1327a90020ca8c/integration/helpers/constants.go#L9)